src/main/gov/nasa/jpf/jdart/JDart.java

```diff
@@ -257,6 +257,7 @@ public String format(LogRecord record) {
         
         //logger.info("Initial valuation: ", ca.getInitialValuation());
         if (!config.getBoolean("jdart.tree.dont.print")) {
+          ca.getConstraintsTree().toJson();
```

Comment:
Yeah, I have thought about make it option to output to json. I can make it as an option.